### DISCUSSION TITLE
[Show]: A Tip if Using Embla with Storybook

### DISCUSSION THREAD

**Discussion by manavm1990** [davidjerleke/embla-carousel#781] [General]
### Summary

Embla fails to behave properly when 🏃🏾‍♂️  running inside of a Storybook Story...

### Additional information

...unless one includes this in their `.stories`:

```tsx
parameters: {
    layout: "fullscreen",
  },
```

Failure to establish a full screen seems to really confuse Embla and is just generally very frustrating.

As soon as I discovered this, I was back to being productive.

**Comment by davidjerleke** [comment]
@manavm1990 thanks 👍. I hope this tip helps other devs.

**As for the problem:** You didn't share any proof that Embla is the actual cause of the problems in this case. Could be related to Embla of course but may also be related to some stuff Storybook is doing under the hood. Another possibility is that there's something in your code causing this. I don’t think we should jump to conclusions if we don't know exactly what's causing this.

**Reply by manavm1990** [reply]
Fair. Being that it was literally a one-line fix and all worked fine, I left it there.

It could very well be how Storybook is rendering.

Again, with such a simple resolution and limited time, it didn't seem worth it to go deeper in this case.

**Reply by davidjerleke** [reply]
@manavm1990 thanks for clarifying. Regardless of what’s causing this: Thanks for sharing your solution 👍🏻!
